backend/app/agents/shared/classification_router.py

```
@@ -0,0 +1,189 @@
+            if json_start != -1 and json_end != -1:
+                json_str = response[json_start:json_end]
+                parsed = json.loads(json_str)
+                return parsed
+            else:
+                raise ValueError("No JSON found in response")
+
+        except Exception as e:
+            logger.error(f"Error parsing LLM response: {str(e)}")
+            return self._fallback_classification(original_message)
+
+    def _fallback_classification(self, message: str) -> Dict[str, Any]:
```

Pylint (3.3.7)

[warning] Unused argument 'message'

(W0613)